frontend/src/container/QueryTable/QueryTable.tsx

```diff
@@ -10,17 +15,35 @@ export function QueryTable({
 	renderActionCell,
 	modifyColumns,
 	renderColumnCell,
+	sorter,
 	...props
 }: QueryTableProps): JSX.Element {
+	const createSorter: CreateSorter = useCallback(
+		(column: DynamicColumn): QueryTableProps['sorter'] => {
+			if (sorter) {
```

Comment:
this would add the sorter in the all the columns to make scalable can we have control this from columns keys vs sorter ?